lib/rules/jsx-no-literals.js

```diff
@@ -485,7 +447,7 @@ module.exports = {
               });
           });
       },
-    } : false, {
+    } : {}, {
```

Comment:
why this change? primitives are object-spread the same as `{}`.

Reply:
I don't remember why I did this change exactly, probably because `false` isn't compatible with the latest TypeScript or `@types/eslint`. 

Anyway, let me revert this change for now. We might will need to change it to `{}` in the future if we update TypeScript or `@types/eslint`.